src/ExternalAPI/API.cs

```diff
@@ -81,7 +82,7 @@ public static GameObject GetGameObject(string uuid)
             }
         }
 
-        public static string PlaceStatic(string modelName, string bodyName, double lat, double lng, float alt, float rotation, bool isScanable = false)
+        public static string PlaceStatic(string modelName, string bodyName, double lat, double lng, float alt, float rotation, bool isScanable = false, string groupname = "SaveGame")
```

Comment:
You can't add optional parameters to API methods. This would break all consumers until they recompile against the new version of KK. Need to create a new overload for the method.

Reply:
Oh, ok. Didn't knew that, thanks.
But I don't think a new overload is necessary because the API class was not public, so no one could use this function anyway.